cookbooks/fb_systemd/recipes/udevd.rb

```diff
@@ -18,19 +18,26 @@
 # limitations under the License.
 #
 
+udevadm = value_for_platform(
+  'centos' => {
+    '< 6.0' => '/sbin/udevadm'
+  },
+  default => '/bin/udevadm',
```

Comment:
`default` needs to be a string for this to work

Reply:
> `default` needs to be a string for this to work

Fixed.